bot/exts/evergreen/timed.py

```diff
@@ -0,0 +1,36 @@
+from copy import copy
+from time import perf_counter
+
+from discord import Message
+from discord.ext import commands
+
+
+class TimedCommands(commands.Cog):
+    """Time the command execution of a command."""
+
+    @staticmethod
+    async def create_execution_context(ctx: commands.Context, command: str) -> commands.Context:
+        """Get a new execution context for a command."""
+        msg: Message = copy(ctx.message)
+        msg._update({"content": f"{ctx.prefix}{command}"})
+
+        return await ctx.bot.get_context(msg)
+
+    @commands.command(name="timed", aliases=["t"])
+    async def timed(self, ctx: commands.Context, *, command: str) -> None:
+        """Time the command execution of a command."""
+        new_ctx = await self.create_execution_context(ctx, command)
+
+        if new_ctx.command and new_ctx.command.qualified_name == "timed":
+            return await ctx.send("You are not allowed to time the execution of the `timed` command.")
```

Comment:
The return annotation is `None`, but you're returning a `Message` object.
```suggestion
            await ctx.send("You are not allowed to time the execution of the `timed` command.")
            return
```